.github/workflows/release-drafter.yml

```diff
@@ -2,15 +2,20 @@ name: Release Drafter
 
```

Comment:
Pls rename the workflow & the file adequately

Reply:
done in the newest commit